src/main/webapp/features/screens/game/room/ct-game-room.html

```diff
@@ -0,0 +1,45 @@
+<div class="row">
+
+    <div ng-if="game.isLoaded()">
```

Comment:
to avoid empty fields when template loaded before we receive http response from server? In such case maybe we can show some progress bar or await on all pages, when are waiting for some http response from API calls? but we can leave it as is now